packages/template-retail-react-app/app/hooks/use-active-data.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {ACTIVE_DATA_ENABLE} from '@salesforce/retail-react-app/app/constants'
+
+const useActiveData = () => {
+    return {
+        async sendViewProduct(category, product, type) {
+            if (ACTIVE_DATA_ENABLE == 0) {
```

Comment:
After converting to `ACTIVE_DATA_ENABLE = true` we can avoid the suboptimal usage of `==` which is not a good comparator in javascript
```suggestion
            if (!ACTIVE_DATA_ENABLE) {
```